docs/applications.md

```diff
@@ -126,7 +126,7 @@ export function unload(props) {
 ```
 
 ## Passing custom properties to child apps
-Each lifecycle method has a `props` property which contains an object with a `name` string and a `customProps` object wich holds any information you want to pass to your child app.
+Each lifecycle method has a `props` property which contains an object with a `name` string, a `mountParcel` [function]('/docs/parcel-api#mountParcel'), and all additional props pass to your child app.
```

Comment:
> and all additional props pass to your child app

Incorrect grammar ^

Maybe:
> and all custom props that were passed to the application.

(Getting rid of child app terminology, too, since we no longer use that term)